source/common/http/header_map_impl.cc

```diff
@@ -18,6 +18,8 @@
 namespace Envoy {
 namespace Http {
 
+bool HeaderStringValidator::disable_validation_for_tests_ = false;
```

Comment:
Could we set this to false in envoy/http/header_map.h, or does that not work because c++?

Reply:
Yes, C++ can only initiailize `const` in the class declaration.